src/devices/CharacterLcd/Font5x7.cs

```diff
@@ -0,0 +1,10186 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// <autogenerated />
+using System.Collections.Generic;
```

Comment:
how is this data generated? any chance we could generate this on the fly rather than having this dictionary?

Reply:
it is generated out of bdf files, various ones but we will have to adjust all this. Most of the characters are not the proper size. In all cases, a similar size or even large file will have to be included. It can be as well a raw stack of bytes, that doesn't really matter.
There is still work here to go because of the size that is not exactly what we need.

Reply:
Can this use BdfFont similarly as LedMatrix rather than including this?

Reply:
We can try problem is that the bdf fonts are not proper ones. The one 5x7 is in reality 5x6, we're looking for a real 5x7 and real 5x8 with all proper unicode. That's the key challenge so far.

Reply:
And I'm all ok to, remove the font and the dome associated to it part of the program.cs and move forward this binding. We can work in parallel on this font issue. Just let me know (I won't have much time this week anyway)